gvsbuild/patches/pango/001-disable-dwrite.patch

```diff
@@ -0,0 +1,11 @@
+--- pango-1.52.1/meson.build.orig	2024-03-22 07:35:38.070249300 +0100
++++ pango-1.52.1/meson.build	2024-03-22 07:35:48.181258400 +0100
+@@ -377,7 +377,7 @@
+   if cairo_dep.found()
```

Comment:
shouldn't we remove the entire `if` at this point?

Reply:
I can change it, however I just ported over the MSYS2 patch as is.